cloud/deploys/bin/azure/saml-keystore-setup

```diff
@@ -0,0 +1,64 @@
+#! /usr/bin/env bash
```

Comment:
suggestion (blocking): this file is generic to anything using SAML auth, please move it to `cloud/azure/bin`